gap/SnAnUnknownDegree.gd

```diff
@@ -0,0 +1,12 @@
+DeclareGlobalName("AllPrimesUpTo");
```

Comment:
Don't use `DeclareGlobalName`, it doesn't work in GAP <= 4.11.

Might be best to just drop this file?

Reply:
Done